scripts/h5peditor-av.js

```diff
@@ -318,11 +318,14 @@ H5PEditor.widgets.video = H5PEditor.widgets.audio = H5PEditor.AV = (function ($)
     var defaultQualityName = H5PEditor.t('core', 'videoQualityDefaultLabel', { ':index': index + 1 });
     var qualityName = (file.metadata && file.metadata.qualityName) ? file.metadata.qualityName : defaultQualityName;
 
-    // Check if source is provider (Vimeo, YouTube, Panopto)
-    const isProvider = file.path && C.findProvider(file.path);
+    // Check if source is YouTube
+    var youtubeRegex = C.providers.filter(function (provider) {
+      return provider.name === 'YouTube';
+    })[0].regexp;
+    var isYoutube = file.path && file.path.match(youtubeRegex);
 
     // Only allow single source if YouTube
-    if (isProvider) {
```

Comment:
What is the purpose of this change? Why do we limit it to youtube?

Reply:
Not sure, that comes with the plugin, i was matching the version that we have (uploaded in the server) with the version that the respository has!

Reply:
@schup Should we fork from the last version ?

Reply:
I believe we are up-to-date on the master branch.
![image](https://user-images.githubusercontent.com/112154/228257720-5eeabf45-94c0-4cf0-81c6-433413937fb1.png)